source/Cosmos.HAL2/Drivers/PCI/Video/DoubleBufferedVMWareSVGAII.cs

```diff
@@ -0,0 +1,703 @@
+﻿using Cosmos.Core;
+using System;
+
+namespace Cosmos.HAL.Drivers.PCI.Video
+{
+    /// <summary>
+    /// VMWareSVGAII class.
+    /// </summary>
+    public class DoubleBufferedVMWareSVGAII
+    {
+        #region Parameter
+
+        /// <summary>
+        /// Register values.
+        /// </summary>
+        public enum Register : ushort

```

Comment:
The following types/definitions have no reason to be duplicated by the other vmare svgaii driver.